Add layout and spacing documentation (#169)
RockefellerArchiveCenter/styles: stories/global/layout/Layout.stories.mdx

import * as gridLayoutStories from './GridLayout.stories.js';

<Meta title="Global/Layout" />

# Layout
Layout dictates how components appear on the page in relation to each
other, including on different devices and screen sizes.

## Usage
Use [CSS Flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox),
[CSS Grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout), and the `float` property as needed to
position components and set page areas. There are multiple page wrapper and layout helper classes to
facilitate layout:

- `container`: Sets a container elements with 15px margin gutters on either side of the
container with a width of 100% minus the gutters.
- `container--full-width`: Sets a container element to `width: 100%` with no gutters.
- `wrapper`: Creates a flexbox wrapper (`display: flex`) that centers the content inside
with  `justify-content: center;`
- `flex`: Simply adds `display: flex` to an element.
- `grid`: Creates a CSS Grid container with 12 columns that fill the container by applying
  the styles `display: grid` and `grid-template-columns: repeat(12, 1fr)` to the element.

## Best practices

> Design for a responsive layout that includes small screens using a single-column

Reviewer: ```suggestion
Design for a responsive layout by initially creating styles for small screens using a single-column
```

> layout. Use media queries as needed to adjust for medium screens and

Reviewer: ```suggestion
layout. Then, enhance using media queries as needed to adjust for medium screens and
```
